cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -515,7 +515,7 @@ users:
 					AlwaysAllowPaths:             []string{"/healthz"}, // note: this does not match /healthz/ or /healthz/*
 				},
 			},
-			expectedUsername: "c1",
+			expectedUsername: []string{"c1", "c2"},
```

Comment:
We are not actively supporting multiple kubeconfig from command line. Can this test be disabled instead?